client_app/src/App.js

```diff
@@ -1,14 +1,38 @@
-import {UpcomingShifts} from "./Shifts";
+import { BrowserRouter as Router, Routes, Route, Link } from "react-router-dom";
+
+import logo from "./Assets/gethelplogo.png";
+
+import { PastShifts, UpcomingShifts } from "./Shifts";
 import Shelters from "./Shelters";
+import VolunteerDashboard from "./VolunteerDashboard";
+import ShelterDashboard from "./ShelterDashboard";
```

Comment:
I think we don't want shelter dashboard in this app, since this is a volunteer facing app. So just volunteer dashboard.